ShoppingCart/CartItem.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ShoppingCart
+{
+    public class CartItem
+    {
+        public Product Product;
```

Comment:
use private setter for these properties

Reply:
For quantity, I am setting in cart handler post product addition/deletion